docs/sources/flow/reference/components/pyroscope.scrape.md

```diff
@@ -15,12 +15,35 @@ title: pyroscope.scrape
 
 {{< docs/shared lookup="flow/stability/beta.md" source="agent" version="<AGENT VERSION>" >}}
 
-`pyroscope.scrape` configures a [pprof] scraping job for a given set of
-`targets`. The scraped performance profiles are forwarded to the list of receivers passed in
-`forward_to`.
+`pyroscope.scrape` collects [pprof] performance profiles for a given set of HTTP `targets`. 
+
```

Comment:
I usually share this picture to show visually what the grafana agent does... may be worth adding in here:

![image](https://github.com/grafana/agent/assets/23323466/c9829f16-3c6a-47e7-b896-0f9767b75adc)

Reply:
Since the picture shows the interactions between different components, I'd prefer not to place it in the reference docs for pyroscope.scrape. It'd be great to have a new "Collecting Profiling Go Applications" task-based doc in the [Getting Started](https://grafana.com/docs/agent/next/flow/getting-started/) section. That doc can walk people through installing the Agent for profiling and we could put the picture there.

I think having such an architecture picture would be great for most task-based docs, and it would be even better if they are all consistent.
cc @clayton-cornell